packages/base/src/commands.ts

```diff
@@ -1085,27 +1085,35 @@ export function addCommands(
       commands.notifyCommandChanged(CommandIDs.transform);
     }
   });
-
+  let toggled: boolean;
   commands.addCommand(CommandIDs.updateCameraSettings, {
-    label: trans.__('Camera Settings'),
+    label: trans.__('Choose between Perspective and Orthographic Projection.'),
     isEnabled: () => Boolean(tracker.currentWidget),
-    icon: cameraSolidIcon,
+    icon: videoSolidIcon,
+    isToggled: () => {
+      const current = tracker.currentWidget;
+      if (current) {
+        const content = current?.content;
+        const projection = content['Type'];
+        console.log('projection:', projection);
+        return (toggled = projection === 'Perspective');
+      } else {
+        return false;
+      }
+    },
     execute: async () => {
+      toggled = !toggled;
```

Comment:
We shouldn't use a global `toggled` variable here, instead we should check the current projection type in the current document.